src/reactor.cc

```diff
@@ -0,0 +1,402 @@
+#include "reactor.h"
+
+#include "boost/shared_ptr.hpp"
+
+namespace tricycle {
+
+Reactor::Reactor(cyclus::Context* ctx) : cyclus::Facility(ctx) {
+  // capacities set somewhat arbitrarily
+  fuel_tracker.Init({&tritium_reserve}, 1000.0);
+  blanket_tracker.Init({&blanket}, 100000.0);
+  excess_tritium_tracker.Init({&tritium_storage}, 1000.0);
+  helium_tracker.Init({&helium_storage}, 1000.0);
```

Comment:
magic numbers.  Should these be user inputs?

Reply:
I can't make them user inputs (the inventory trackers have to be initialized in the constructor, which it looks like goes before the user inputs are processed). I just made them member variables. They're meant to be ceilings on values anyway, so they're just set to sort of unreasonably high numbers.